cypress/e2e/pages/mediaAssetPage/index.cy.js

```diff
@@ -1,11 +1,353 @@
-import runTestsForPage from '../../../support/helpers/runTestsForPage';
-import { testsThatFollowSmokeTestConfigForAMPOnly } from './testsForAMPOnly';
-import { testsThatFollowSmokeTestConfigForCanonicalOnly } from './testsForCanonicalOnly';
+import runTestsForPage from '#nextjs/cypress/support/helpers/runTestsForPage';
+import { testsThatAlwaysRunForAllPages as testsForAllPages } from '../testsForAllPages';
+import { testsThatFollowSmokeTestConfigForAllCanonicalPages as testsForAllCanonicalPages } from '../testsForAllCanonicalPages';
+import { testsThatFollowSmokeTestConfigForAllAMPPages as testsForAllAMPPages } from '../testsForAllAMPPages';
+// import canonicalAndAmpArticleTests from './tests';
+import ampArticleTests from './testsForAMPOnly';
+import canonicalArticleTests from './testsForCanonicalOnly';
+// import getOptimizelyKey from '../../../support/helpers/getOptimizelyKey';
```

Comment:
Let's remove this - we're unlikely to run experiments on the old media asset page
```suggestion
```